app.js

```diff
@@ -9,7 +9,8 @@ const formatsLogger = app.get("env") === "development" ? "dev" : "short";
 app.use(logger(formatsLogger));
 app.use(cors());
 app.use(express.json());
-app.use("/api/contacts", require("./api"));
+app.use("/api/contacts", require("./api/contacts"));
```

Comment:
Zauważyłem że dodajesz auth wszędzie w contacts route. Można się zastanowić czy nie dodać tego lepiej tu. Piszę to bardziej żeby się zastanowić niekoniecznie zmieniać.

Reply:
Czyli wystarczy w app.js dodać: app.use("/api/auth", require("./middlewares/auth")); zamiast w routes?

Reply:
```suggestion
app.use("/api/contacts", auth, require("./api/contacts"));
```
Takie coś sprawi że middleware działa na wszystkich ścieżkach contacts.
Ps. ja nie zawsze widzę komentarze tutaj, wiec jakbym nie pisał to napisz na slack/lms ze potrzebujesz żebym się odniósł .

Reply:
ok, dlatego właśnie na LMS napisałam :)